application/services/Reports.php

```diff
@@ -6515,7 +6517,109 @@ public function getXtptIndicatorsReport($params) {
                 }
             }
         }
+        // Add paticipants require attention Sheet
+        $attentionRequiredParticipantSheet = new PHPExcel_Worksheet($excel, "");
+        $excel->addSheet($attentionRequiredParticipantSheet, $sheetIndex);
+        $sheetIndex++;
+        $sheetIndex++;
+        $attentionRequiredParticipantSheet->getCellByColumnAndRow(0, 1)->setValueExplicit(html_entity_decode("Participant That Require Attention from their PTcC ", ENT_QUOTES, 'UTF-8'), PHPExcel_Cell_DataType::TYPE_STRING);
```

Comment:
Please change this to just "Participants Requiring Attention"

Reply:
ok